pkg/dom0-ztools/Dockerfile

```diff
@@ -1,9 +1,24 @@
 ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:6.7.0
 # hadolint ignore=DL3006
 FROM ${EVE_BUILDER_IMAGE} as zfs
+ENV BUILD_PKGS git attr-dev e2fsprogs-dev zlib-dev libtirpc-dev openssl-dev util-linux-dev \
+    automake autoconf libtool linux-headers build-base
 ENV PKGS zfs ca-certificates util-linux
 RUN eve-alpine-deploy.sh
 
+# building of zvol_id may be skipped if we will add zfs-udev to eve-alpine
+ENV ZFS_VERSION=2.0.4
+ENV ZFS_COMMIT=zfs-${ZFS_VERSION}
+ENV ZFS_REPO=https://github.com/zfsonlinux/zfs.git
+
+WORKDIR /tmp/zfs
+RUN git clone --depth 1 -b ${ZFS_COMMIT} ${ZFS_REPO} .
+
+RUN ./autogen.sh && \
```

Comment:
I am curious -- is the required bits not packaged with the Alpine (maybe latest Alpine?). I mean -- I'm OK rebuilding it -- but it seems like a bit of a waste @giggsoff

Reply:
It exists in zfs-udev package, but it is not inside eve-alpine. Should I open another PR for adding the package into eve-alpine and then modify this PR with new EVE_BUILDER_IMAGE hash?

Reply:
Opened https://github.com/lf-edge/eve/pull/2159

Reply:
Adding zfs-udev to eve-alpine is totally cool @giggsoff -- I'm just not quite sure why you need to bump the dependency all the way to Alpine 3.14?

Reply:
Rebased on new eve-alpine and removed building step